### PR TITLE
Viz tutorial

### PR DESCRIPTION
**Pull request by mlautman**
updating screenshots and edits to visualization_tutorial.rst

### CODE REVIEW

File: doc/ros_visualization/visualization_tutorial.rst [hunk 1]
```diff
@@ -142,16 +144,15 @@ Moving out of reachable workspace
 +++++++++++++++++++++++++++++++++
 
 Note also what happens when you try to move an end-effector out of its
-reachable workspace (sometimes the access denied sign will not
-appear).
+reachable workspace.
```
Comment (davetcoleman) [inline doc/ros_visualization/visualization_tutorial.rst]: ive never seen this access denied sign, i wonder what happened to it??

Comment (davetcoleman) [reply doc/ros_visualization/visualization_tutorial.rst]: can you try to restore this mesh asset?
https://github.com/ros-planning/moveit/blob/e97038d476c4e45817e80f6ec5e58d5840ccf004/moveit_ros/robot_interaction/src/interactive_marker_helpers.cpp#L118

Comment (mlautman) [reply doc/ros_visualization/visualization_tutorial.rst]: Turns out it's broken beyond adding the DAE back in.

**Comment by davetcoleman** [comment]
please delete your branches after merging

**Pull request merged**